features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/internal/TracingFeature.kt

```diff
@@ -73,4 +78,17 @@ internal class TracingFeature constructor(
             internalLogger = internalLogger
         )
     }
+
+    private fun createOtelDataWriter(
+        sdkCore: FeatureSdkCore
+    ): com.datadog.trace.common.writer.Writer {
+        val internalLogger = sdkCore.internalLogger
+        return OtelTraceWriter(
+            sdkCore,
+            legacyMapper = OtelDdSpanToSpanEventMapper(networkInfoEnabled),
```

Comment:
If we are creating a new writer, why do we have the notion of `legacy` here?

Reply:
not sure why I added this back then I guess it is more `DdSpanToSpanEvent` mapper